USAGE.md

```diff
@@ -14,11 +14,9 @@ is not guaranteed.
 
 ## Installation
 
-Currently only available for the Nitrokey 3A Mini.
-
-Download the latest compiled [release](https://github.com/Nitrokey/opcard-rs/releases) ZIP file.
-Plug your Nitrokey 3A Mini and use [nitropy](https://docs.nitrokey.com/software/nitropy/) to install it with 
-`nitropy nk3 update <path/to/release/zip/file>`
+Download the latest compiled [release](https://github.com/Nitrokey/opcard-rs/releases).
+Plug your Nitrokey 3 and use [nitropy](https://docs.nitrokey.com/software/nitropy/) to install it with 
+`nitropy nk3 update <path/to/release/file>`
```

Comment:
I think this should refer to the NK3 alphas instead as we no longer have binaries for opcard.